tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/ContactSupportCapabilitiesModule.scala

```diff
@@ -0,0 +1,32 @@
+package com.linagora.tmail.james.jmap
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.ProvidesIntoSet
+import com.linagora.tmail.james.jmap.method.CapabilityIdentifier.LINAGORA_CONTACT_SUPPORT
+import jakarta.inject.Inject
+import org.apache.james.core.MailAddress
+import org.apache.james.jmap.core.CapabilityIdentifier.CapabilityIdentifier
+import org.apache.james.jmap.core.{Capability, CapabilityFactory, CapabilityProperties, UrlPrefixes}
+import play.api.libs.json.{JsObject, Json}
+
+case class ContactSupportProperties(mailAddress: MailAddress) extends CapabilityProperties {
+  override def jsonify(): JsObject = Json.obj("supportMailAddress" -> mailAddress.asString())
+}
+
+case class ContactSupportCapability(contactSupportProperties: ContactSupportProperties) extends Capability {
+  val properties: CapabilityProperties = contactSupportProperties
+  val identifier: CapabilityIdentifier = LINAGORA_CONTACT_SUPPORT
+}
+
+case class ContactSupportCapabilityFactory @Inject()(jmapConfig: JMAPExtensionConfiguration) extends CapabilityFactory {
+  override def create(urlPrefixes: UrlPrefixes): Capability =
+    ContactSupportCapability(ContactSupportProperties(new MailAddress(s"hnasri-${jmapConfig.publicAssetTotalSizeLimit}@linagora.com")))
+
+  override def id(): CapabilityIdentifier = LINAGORA_CONTACT_SUPPORT
+}
+
+class ContactSupportCapabilitiesModule() extends AbstractModule {
+
+  @ProvidesIntoSet
+  private def capability(jmapConfig: JMAPExtensionConfiguration) = ContactSupportCapabilityFactory(jmapConfig)
```

Comment:
```suggestion
    override def configure(): Unit = {
    Multibinder.newSetBinder(binder(), classOf[CapabilityFactory])
      .addBinding()
      .to(classOf[ContactSupportCapabilityFactory])
  }
```
Then `MemoryLinagoraContactSupportCapabilityContract#shouldReturnContactSupportCapability` will green

Reply:
Thanks! Now I need to understand why there were no logs printed before.

Reply:
because it is no error
it just no show contact capability in the session route.
I added `.prettypeak` in your test and observed that.

Reply:
`@ProvidesIntoSet` should have registered the capability factory, right?

Reply:
Not tested
But may be it should be:
```scala
@ProvidesIntoSet
  private def capability(jmapConfig: JMAPExtensionConfiguration): CapabilityFactory = ContactSupportCapabilityFactory(jmapConfig)
```

Reply:
One more point
if you like `@ProvidesIntoSet` 
it can be :
```suggestion
  @ProvidesIntoSet
  private def capability(contactSupportCapabilityFactory: ContactSupportCapabilityFactory) : CapabilityFactory = contactSupportCapabilityFactory
```
// the `ContactSupportCapabilityFactory` already has `@Inject` anotation